cmake/external/extensions.cmake

```diff
@@ -11,9 +11,18 @@ set(OCOS_ENABLE_CTEST OFF CACHE INTERNAL "")
 set(OCOS_ENABLE_STATIC_LIB ON CACHE INTERNAL "")
 set(OCOS_ENABLE_SPM_TOKENIZER OFF CACHE INTERNAL "")
 
+# backup CMAKE_CXX_FLAGS in case we'll rewrite it for exceptions flag
+set(CMAKE_CXX_FLAGS_BAK "${CMAKE_CXX_FLAGS}")
+
```

Comment:
Is this required? I thought CMake flags were scoped.

Reply:
I think so.
The [DOC](https://cmake.org/cmake/help/latest/command/include.html) says "Variable reads and writes access the scope of the caller (dynamic scoping".
and I also verified in my local environment, the "include" instruction does changed the caller's var of CMAKE_CXX_FLAGS.